{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "134eca34",
   "metadata": {},
   "outputs": [],
   "source": [
    " #!/bin/bash\n",
    " # Phenotype counts\n",
    " plink --bfile dent1 --freq --missing --make-just-fam --out QC_summary\n",
    " \n",
    " # Sex Check\n",
    " plink --bfile dent1 --check-sex --out QC0\n",
    " \n",
    " # Run Script 1 in R\n",
    " #Remove Problem\n",
    " plink --bfile dent1 --remove QC0.problem --make-bed --out data.afterQC0\n",
    " \n",
    " # Missingness (Individuals)\n",
    " plink --bfile data.afterQC0 --missing --out QC1\n",
    " \n",
    " # Run script 2\n",
    " # Remove bad ids\n",
    " plink --bfile data.afterQC0 --remove QC1.badids --make-bed --out data.afterQC1\n",
    " \n",
    " # Missingness (SNPs)\n",
    " plink --bfile data.afterQC1 --missing --out QC2\n",
    " \n",
    " # Run Script 3\n",
    " #Remove bad SNPs\n",
    " plink --bfile data.afterQC1 --exclude QC2.badsnp --make-bed --out\n",
    " data.afterQC2\n",
    " \n",
    " # LD Pruning\n",
    " plink --bfile data.afterQC2 --indep-pairwise 50 5 0.2 --out data.r02\n",
    "sh extract_SNPs_LRLDregions.sh\n",
    " plink --bfile data.afterQC2 --extract data.r02.prune.in --exclude\n",
    " longrangeLDregionSNPs.txt --make-bed  --out data.afterQC2.r02\n",
    "\n",
    " # Relatedness\n",
    " plink --bfile data.afterQC2.r02 --genome --min 0.2 --out data.IBD\n",
    " \n",
    " # PCA Round 1\n",
    " plink --bfile data.afterQC2.r02 --pca header tabs --out data.pca\n",
    " \n",
    " # Run Script 4\n",
    " #PCA Round 2\n",
    " plink --bfile data.afterQC2.r02 --maf 0.05 --keep QC3.pcaids --pca header tabs--out data.pca2\n",
    " \n",
    " # Run Script 5\n",
    " # Heterozygosity\n",
    " plink --bfile data.afterQC2 --het --keep QC3.pcaids  --out QC4\n",
    " \n",
    " # Script 6\n",
    " #HWE\n",
    " plink --bfile data.afterQC2 --keep QC3.pcaids --hwe midp 10e-6 --make-bed --\n",
    " out data.clean\n",
    " # GWAS\n",
    " plink --autosome --bfile data.clean --logistic --hide-covar --covar\n",
    " data.pca2.eigenvec --parameters 1-10 --out data.gwas"
   ]
  }
 ],
 "metadata": {
  "language_info": {
   "name": "python"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
